modules/gapi/src/api/freetype_mask_creator.hpp

```diff
@@ -113,14 +104,22 @@ class FreeTypeBitmaskCreator : public IBitmaskCreator
 
     virtual ~FreeTypeBitmaskCreator() override
     {
-        FT_Done_Face(m_face);
-        FT_Done_FreeType(m_library);
+        GAPI_Assert(!FT_Done_Face(m_face));
+        GAPI_Assert(!FT_Done_FreeType(m_library));
```

Comment:
Never throw from destructor! That's harmful.

Reply:
Fixed